MyReadingList/MyReadingList/MyReadingListView.swift

```diff
@@ -10,21 +10,19 @@ import SwiftUI
 
 struct MyReadingListView: View {
     var body: some View {
-        VStack {
-            Text(NSLocalizedString("ui.myReadingList.title", value: "My Reading List", comment: "title for the MyReadingList screen"))
-                .font(.headline)
-            Spacer()
-            
-            List {
-                HStack{
-                    Button {
-                        //do nothing for now
+        NavigationView {
+            VStack {
```

Comment:
Is this VStack necessary?

Reply:
In the begin I thought I will need both a VStack and HStack to achieve my task, but ready more about it. I learn that there is no need of VStack. That's why I will remove it in the next commit.

Reply:
Sounds good!